website/docs/docs/dbt-versions/release-notes/04-Sept-2023/ci-updates-phase2-rn.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Update: Improvements to dbt Cloud continuous integration"
+description: "September 2023: dbt Cloud now has two types of jobs &mdash; deploy jobs and CI jobs &mdash; with streamlined setup and improved efficiency. "
+sidebar_label: "Update: Improvements to dbt jobs"
+tags: [Sept-2023, CI]
+date: 2023-09-11
+sidebar_position: 10
+---
+
+dbt Cloud now has two distinct job types: [deploy jobs](/docs/deploy/deploy-jobs) for building production data assets, and [CI jobs](/docs/deploy/ci-jobs) for checking code changes. These jobs perform fundamentally different tasks so dbt Labs improved the setup experience with better defaults for each. 
+
+Since there are now two types of jobs, instead of one generic type, we can better guide you through the setup flow. Best practices are built into the default settings so you can go from curious to being set up in seconds.
```

Comment:
```suggestion
With two types of jobs, instead of one generic type, we can better guide you through the setup flow. Best practices are built into the default settings so you can go from curious to being set up in seconds.
```